.changeset/rare-cameras-bow.md

```diff
@@ -0,0 +1,56 @@
+---
+'@commercetools-frontend/application-components': minor
+'@commercetools-local/visual-testing-app': minor
+'@commercetools-website/custom-applications': minor
+'@commercetools-website/components-playground': minor
+---
+
+Add 2 new components: `<TabularMainPage>` and `<TabularDetailPage>` providing navigation through, for instance, `<TabHeader>` components. Both pages, as opposed to `<TabularModalPage>`, are not rendered within a modal.
```

Comment:
I would also mention where these pages are supposed to be used.

Reply:
Updated in cdb034d. However, I'm not sure if what I wrote is what you had in mind.